content/en/_index.md

```diff
@@ -201,6 +281,18 @@ possible, provide a link to documentation that enumartes the options.
         exampleSnippet(projectId, filePath);
     }
   {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    function main() {
+      // Describe what arg1 does
+      const arg1 = ...
+
+      // Describe what arg2 does
+      const arg2 = ...
+      ...
+
+      exampleSnippet(arg1, arg2, ...);
+    }
```

Comment:
I think this is better understood with an actual example, like the Java example uses:

```suggestion
    function main() {
      // TODO(developer): Replace these variables before running the sample.
      const projectId = "my-project-id";
      const filePath = "path/to/image.png";
      exampleSnippet(arg1, arg2);

      exampleSnippet(arg1, arg2, ...);
    }
```

Reply:
OK - done.